batchflow/utils_transforms.py

```diff
@@ -20,14 +21,18 @@ class Normalizer:
     normalization_stats : dict, optional
         If provided, then used to get statistics for normalization.
         Otherwise, compute them from a given array.
+    process_nan : bool
+        Whether to process nan values in statistics evaluation or not.
+        `np.nan*`/`bn.nan*` functionality is slower than default statistics computation, but sometimes it is necessary.
     """
 
     def __init__(self, mode='meanstd', clip_to_quantiles=False, q=(0.01, 0.99),
-                 normalization_stats=None):
+                 normalization_stats=None, process_nan=False):
         self.mode = mode
         self.clip_to_quantiles = clip_to_quantiles
         self.q = q
         self.normalization_stats = normalization_stats
+        self.process_nan = process_nan
```

Comment:
I dislike `process_nan` as they will be processed in either case. A consistent name would be `propagate_nan`. I would prefer `expect_nan`. What do you think?